# HG changeset patch
# User twisti
# Date 1400640658 25200
#      Tue May 20 19:50:58 2014 -0700
# Node ID cb9aea9e6eb75c7f1ca7b2dd477b48d481261fa7
# Parent  9664162a7109691191835f5d625daba3ce5f78a5
8042603: 'SafepointPollOffset' was not declared in static member function 'static bool Arguments::check_vm_args_consistency()'
Reviewed-by: kvn

diff -r 9664162a7109 -r cb9aea9e6eb7 src/share/vm/runtime/arguments.cpp
--- openjdk/hotspot/src/share/vm/runtime/arguments.cpp	Fri May 30 13:32:43 2014 -0400
+++ openjdk/hotspot/src/share/vm/runtime/arguments.cpp	Tue May 20 19:50:58 2014 -0700
@@ -2342,7 +2342,9 @@
   status &= verify_interval(NmethodSweepFraction, 1, ReservedCodeCacheSize/K, "NmethodSweepFraction");
   status &= verify_interval(NmethodSweepActivity, 0, 2000, "NmethodSweepActivity");
 
+#ifdef COMPILER1
   status &= verify_interval(SafepointPollOffset, 0, os::vm_page_size() - BytesPerWord, "SafepointPollOffset");
+#endif
 
   return status;
 }
